fr.esrf.tango.generator.xtend/src/fr/esrf/tango/pogo/generator/python/PythonTypeDefinitions.java

```diff
@@ -319,7 +319,13 @@ public static String defaultValueHL (Attribute attr) {
 		if (attr.getDataType() instanceof EncodedType)			def_val =  "['', '']";
 		if (attr.getDataType() instanceof EnumType)
 		{
-			def_val = toFirstUpper(attr.getName())+ "."+ attr.getEnumLabels().get(0);
+			if(attr.getEnumLabels().get(0).contains("-") || attr.getEnumLabels().get(0).contains("%") || attr.getEnumLabels().get(0).contains("*")
+					|| attr.getEnumLabels().get(0).contains("+") || attr.getEnumLabels().get(0).contains("=")) {
```

Comment:
This could be a lot shorter, and easier to read if the first label was put in a local variable before all the checks.

Reply:
Agree. Will do that change.